packages/core/src/blocks/ImageBlockContent/ImageBlockContent.ts

```diff
@@ -57,6 +57,17 @@ export const imageRender = (
   block: BlockFromConfig<typeof imageBlockConfig, any, any>,
   editor: BlockNoteEditor<any, any, any>
 ) => {
+  // Immediately updates & re-renders the block if it's wider than the editor.
+  if (
```

Comment:
My first feeling is I don't really like this approach:
- what if a user opens the document on a small screen, and later on a large screen?
- what if the user is in read-only mode?